projects/plugins/jetpack/modules/masterbar/admin-menu/class-wpcom-admin-menu.php

```diff
@@ -191,9 +191,15 @@ public function add_stats_menu() {
 
 	/**
 	 * Adds Upgrades menu.
+	 *
+	 * @param string $plan The current WPCOM plan of the blog.
 	 */
-	public function add_upgrades_menu() {
-		parent::add_upgrades_menu();
+	public function add_upgrades_menu( $plan = null ) {
+		if ( class_exists( 'WPCOM_Store_API' ) ) {
+			$products = \WPCOM_Store_API::get_current_plan( get_current_blog_id() );
+			$plan     = $products['product_name_short'];
```

Comment:
In Atomic admin menu class we added a check to see if product_name_short key is present in the products array. Do we need a check here as well? 🤔

Reply:
Good point, done b60bfee